contracts/src/periphery/cert/EigenDACertVerifier.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifier} from "src/periphery/cert/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDACertVerifierBase} from "src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol";
+
+import {IEigenDAThresholdRegistry} from "src/core/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDASignatureVerifier} from "src/core/interfaces/IEigenDASignatureVerifier.sol";
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.sol";
+
+import {EigenDACertVerificationLib as CertLib} from "src/periphery/cert/libraries/EigenDACertVerificationLib.sol";
+
+contract EigenDACertVerifier is IEigenDACertVerifier {
+    error InvalidSecurityThresholds();
+
+    IEigenDAThresholdRegistry internal immutable _eigenDAThresholdRegistry;
+
+    IEigenDASignatureVerifier internal immutable _eigenDASignatureVerifier;
+
+    DATypesV1.SecurityThresholds internal _securityThresholds;
+
+    bytes internal _quorumNumbersRequired;
+
+    uint64 internal constant CERT_VERSION = 3;
```

Comment:
Wdyt about using full semver? I think in general its nice to be able to quickly know what is onchain, and just having the cert_version (major version) makes finding the commit/release that is actually deployed onchain pretty hard I think (unless you have a good solution for this)?

Also if you have some time would be curious for your thoughts on Celo's approach of using a 4th version number for storage: https://eigenlabs.slack.com/archives/C07614NNXDH/p1744966847204929
We prob don't need that but curious in general if its something that you think is useful or not.

Reply:
Also curious why not use `_certVersion` like the other variables?

Reply:
It is just a common solidity convention to use this format for constants.

Reply:
I think a SEMVER can be a separate variable and getter. It might be confusing to see something like SEMVER 3.1 but only see a CertV3 for example.

Reply:
Can this be part of a future PR?

Reply:
Yes will resolve. lets do later.